src/main/java/com/enterprise/fundraisermanager/service/FundraiserServiceStub.java

```diff
@@ -2,12 +2,17 @@
 
 import com.enterprise.fundraisermanager.dao.IFundraiserDAO;
 import com.enterprise.fundraisermanager.dto.Fundraiser;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 import java.util.List;
 
+/**
+ * The service handles business logic and provides fundraiser data to the controller
+ */
 @Service
 public class FundraiserServiceStub implements IFundraiserService{
 
+    @Autowired
     private IFundraiserDAO fundraiserDAO;
```

Comment:
Same here, missing the @Autowired annotation - after controller calls the service the service was also calling methods an a null object.